src/components/style-selector/simple-style-item.vue

```diff
@@ -0,0 +1,16 @@
+<script setup lang="ts">
+const props = defineProps<{
+  colors: string[]
+}>()
+</script>
+
+<template>
+  <div
+    v-for="color in props.colors"
+    :key="color"
+    class="grow m-px"
+    :style="`background-color: ${color}`"
```

Comment:
this is quite difficulat to get to work since there are too many custom colors coming from the configuration. All the values would have to be compiled into tailwind.config.cjs

Reply:
alright